autometa/taxonomy/ncbi.py

```diff
@@ -291,6 +295,8 @@ def parent(self, taxid):
         try:
             taxid = self.convert_taxid_dtype(taxid)
         except DatabaseOutOfSyncError:
+            err_message = f"Databases out of sync. Taxid ({taxid}) not found in nodes.dmp or names.dmp!"
+            logger.error(err_message)
```

Comment:
The `err_message` is already being emitted by `DatabaseOutOfSyncError`. 

I'm not sure if we want to emit a warning or an error here. Probably a warning, since we are continuing on with `taxid = 0`. 

We should probably reserve `logger.error` for any of the entrypoints

```suggestion
        except DatabaseOutOfSyncError as err:
            logger.warning(err)
```

```suggestion
        except DatabaseOutOfSyncError as err:
            logger.error(err)
```